drivers/volume/common.go

```diff
@@ -83,6 +83,10 @@ func (d *DefaultDriver) String() string {
 	return ""
 }
 
+func (d *DefaultDriver) GetVolumeDriverNamespace() (string, error) {
```

Comment:
A bit confused about this, does this always return empty namespace "" and nil error?